src/commands/InspectCommand.ts

```diff
@@ -0,0 +1,77 @@
+import {Message, MessageEmbed} from "discord.js";
```

Comment:
Can you pad all your destructures throughout this, please? For example:
```suggestion
import { Message, MessageEmbed } from "discord.js";
```